app/src/app/components/contents/manageAccess/utils/manageUsersColumns.tsx

```diff
@@ -69,13 +69,20 @@ export const setupManageUsersColumns = (
       return (
         <div className="flex space-x-2 justify-end ">
           {row.original.username === user?.userName ? (
-            <Button variant="outline" size="icon" aria-label="delete-user" disabled>
-              <Trash2 className="h-4 w-4 " />
-            </Button>
+            <>
+              <Button variant="outline" size="icon" aria-label="delete-user" disabled>
+                <Trash2 className="h-4 w-4 " />
+              </Button>
+              <Button variant="outline" size="icon" aria-label="edit-user" disabled>
```

Comment:
I suppose it would be ok for an admin to grant themselves additional permissions (e.g. packet push) but we'd want to stop them from removing permissions. We can probably assume admins gets properly initialised for now though...

Reply:
yeap admin has all the permissions so nothing for them to ad